sycl/test/abi/sycl_symbols_windows.dump

```diff
@@ -1025,7 +1032,6 @@
 ?ext_oneapi_enable_peer_access@device@_V1@sycl@@QEAAXAEBV123@@Z
 ?ext_oneapi_fill2d_impl@handler@_V1@sycl@@AEAAXPEAX_KPEBX111@Z
 ?ext_oneapi_get_default_context@platform@_V1@sycl@@QEBA?AVcontext@23@XZ
-?enable_ext_oneapi_default_context@detail@_V1@sycl@@YAX_N@Z

```

Comment:
Why it is removed? Is it ABI break between the releases?

Reply:
This symbol is moved to another line because  the script was used to generate this file.